CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,118 @@
+# Dolphin Emulator Code of Conduct
+
+Like many Open Source projects and the greater FLOSS community, Dolphin is a
+collaborative open source community comprised of a diverse group of
+contributors and users from around the globe. We find the contributions,
+collaborations, and mentorships within our community to be the essential
+lifeblood of our project and appreciate the efforts of those who participate to
+nurture and grow those, and all other aspects of our community.
+
+However, when a large and sufficiently diverse group of people work together,
+there are often cultural, communication, and compatibility issues. In order to
+minimize conflict, and provide a framework for resolution, we have a brief code
+of conduct that we ask all participants in the Dolphin community adhere to.
+These rules should apply to everyone, regardless of station within the
```

Comment:
This is just a small nitpick, but I don't understand what "station" means in this context.

Reply:
suggest `s/station/status/`

Reply:
For reference that would be definition 6 on https://en.wiktionary.org/wiki/station#Noun, I agree it's not super commonly used in that meaning so I replaced it with @merryhime's suggestion.